subprojects/core/src/main/java/org/gradle/initialization/DefaultSettings.java

```diff
@@ -340,6 +343,12 @@ public BuildCacheConfigurationInternal getBuildCache() {
     @Override
     public void pluginManagement(Action<? super PluginManagementSpec> rule) {
         rule.execute(getPluginManagement());
+
+        List<IncludedBuildSpec> includedBuilds = ((PluginManagementSpecInternal) getPluginManagement()).getIncludedBuilds();
+        for (IncludedBuildSpec buildSpec : includedBuilds) {
+            buildIncluder.registerPluginBuild(buildSpec, gradle);
+            includedBuildSpecs.add(buildSpec);
+        }
```

Comment:
It might  be cleaner/more consistent with other code if "includeBuild" in "DefaultPluginManagementSpec" would directly do "buildIncluder.registerPluginBuild(buildSpec, gradle);".